eng/common/scripts/Package-Properties.ps1

```diff
@@ -281,6 +287,11 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
                 }
 
                 if ($soleCIYml -and $filePath.Replace("`\", "/").StartsWith($directory)) {
+                    #  `sdk/computefleet/Azure.ResourceManager.ComputeFleet/CHANGELOG.md` exists
+                    # and `sdk\compute\ci.mgmt.yml` exists
+                    # does this mean that when `sdk/computefleet/Azure.ResourceManager.ComputeFleet/CHANGELOG.md` changes, the projects under `sdk/compute/ci.mgmt.yml` will be triggered?
+                    # because `sdk/computefleet/Azure.ResourceManager.ComputeFleet/CHANGELOG.md` starts with `sdk/compute`
```

Comment:
No, because we are only entering this code path if it is a changed file at the service directory level. Where service-directory-level is defined by "in the same directory as my owning ci.yml"

Reply:
Scroll up to line 278 just out of view.

Reply:
aah... 
```
 $serviceDirectoryChange = (Split-Path $filePath -Parent).Replace("`\", "/") -eq $directory
  if (!$serviceDirectoryChange) {
      break
  }
  ```
  
This foreach loop only cares about files that are direct siblings of the `*ci*.yml` files from triggeringPaths when there was only 1 `*ci*.yml` in the folder.